apps/accounts/forms/auth.py

```diff
@@ -251,12 +275,18 @@ def clean(self):
                     pass
                 else:
                     self.inactive_user = True
-                    raise forms.ValidationError("That account is not active yet because you haven't confirmed your email. <a id=\"resend_confirmation\" href=\"javascript:void(0);\">Resend email confirmation &gt;</a>")
+                    raise forms.ValidationError(
+                        "That account is not active yet because you haven't confirmed your email. <a id=\"resend_confirmation\" href=\"{}\">Resend email confirmation &gt;</a>".format(  # noqa
+                            reverse('account_resend_confirmation')
```

Comment:
Need to add `HTK_ACCOUNTS_RESEND_CONFIRMATION = 'account_resend_confirmation'` to https://github.com/hacktoolkit/django-htk/blob/master/apps/accounts/constants/defaults.py

And then use the constant here, instead of hardcoding the string.